frontend/components/puzzleA.jsx

```diff
@@ -41,10 +47,9 @@ class PuzzleA extends React.Component {
       return (
         <Button
           key={key}
-          onClick={this.handleOptionClick(question, index)}
           selected={answer === index}
         >
-          <FontAwesomeIcon icon={icon} />
+          <SafeImage src={icon} onClick={this.handleOptionClick(question, index)} />
```

Comment:
why move `onClick` to the image? seems to make more sense on the Button

Reply:
Was trying to mimic a link I was using to teach myself how to display an image. I can move it back.